packages/contracts-bedrock/test/kontrol/scripts/run-kontrol.sh

```diff
@@ -214,7 +214,7 @@ regen=
 max_depth=1000000
 max_iterations=1000000
 smt_timeout=100000
-workers=2
+workers=5
```

Comment:
Can we convert `tests` to an array and set this to `max(arrayLength, someCap)`? That would be easier to maintain

Reply:
See this commit (https://github.com/ethereum-optimism/optimism/pull/9183/commits/1cda84fd382107ac3f7c2fdd7deb4afaca523d85) for the solution. I've set it to `min(arrayLength, 7)` since the CI machine now has 8 cpus. Let me know if any other cap is desirable!

Reply:
Should we just set it to 8 and let kontrol/CircleCI figure out the allocation? Maybe that's not a good idea though, not sure of the tradeoffs.

Also lets add an inline comment here like `// Set to 7 since the CI machine has 8 CPUs`

Reply:
I've left it to 7, with the comment. I've never had Kontrol use up all the cores of my machine, so the cautionary choice seems good to me